code/modules/mob/new_player/sprite_accessories.dm

```diff
@@ -128,6 +128,10 @@
 /datum/sprite_accessory/hair/longpony
 	name = "Long Ponytail"
 	icon_state = "hair_longpony"
+	
+/datum/sprite_accessory/hair/gaypony
+	name = "Gay Ponytail"
```

Comment:
![1590951212594](https://user-images.githubusercontent.com/17928298/83693713-fb861000-a5cc-11ea-9b63-81881bba1d29.jpg)

Reply:
give me a better name and i'll change it